mitxgraders/helpers/calc/expressions.py

```diff
@@ -387,7 +388,12 @@ def get_grammar(self):
 
         # Define + and -
         plus = Literal("+")
-        minus = Literal("-")
+
+        # Also accept unicode emdash
+        emdash = Literal("–") 
```

Comment:
I would MUCH rather use `"\u2014"` here, rather than setting an encoding on the script.

Reply:
Because we're importing unicode_literals, this should work in both python 2.7 and 3.

Reply:
Excellent! Hey! I took 8.02x, it was brilliant, thank you.   About the encoding, yes, I'm happy to make those changes. Sorry for making so much noise with the build, I'm still kicking the tires on codecov and Travis CI. 

Enough excuses from me, hopefully the next commit will be all green.  Let me know if you would prefer that I retract this pull request and squash the previous two commits.

Reply:
Keep the PR - it's got this discussion in it. Once you're all green, we'll squash before merging :-) If you activate a python 2 virtual environment, you can test locally there using pytest.

Reply:
Thanks, I'm all set on testing with python2 now, locally using pyenv with 2.7.15 to match the build bot.

Reply:
There is a small issue outstanding with the new test. I'm not sure how
to fix it. There is a test function found in test_expressions.py
called:

test_negation_with_emdash()

It loops through badformulas testing to make sure badly formatted
formulas are unable to parse.  That bit works, that is if the test
fails then exceptions are raised, however, pytest has a feature that
should be used, but is not in this case. A particular function:
pytest.raises can take a keyword argument called "match" that ensures
error messages raised along with the parse error are formatted
correctly.

There is or was a bug that seems to mirror the issue here:

https://github.com/pytest-dev/pytest/issues/5478